api/models/Post.js

```diff
@@ -1,8 +1,11 @@
 /* globals _ */
+
 import data from '@emoji-mart/data'
 import { init, getEmojiDataFromNative } from 'emoji-mart'
+import insane from 'insane'
```

Comment:
interesting library name

Reply:
This actually shouldn't be being imported here, and doesn't appear to be used anywhere in the file. For keeping our application of sanitization consistent, and not double-sanitizing which causes problems, should be done through using one of the `HyloShared.TextHelpers` methods.